src/Tools/VMC/RecordList.h

```diff
@@ -0,0 +1,100 @@
+////////////////////////////////////////////////////////////////////////
+// \author Christopher Backhouse -- c.backhouse@ucl.ac.uk
+////////////////////////////////////////////////////////////////////////
+
+#ifndef _RECORDLIST_H
+#define _RECORDLIST_H
+
+#include <vector>
+#include <string>
+
+class TFile;
+class TTree;
+
+namespace genie{
+namespace vmc{
+  //---------------------------------------------------------------------------
+  struct Particle
+  {
+    int pdg;
+    float E, px, py, pz;
+  };
+
+  //---------------------------------------------------------------------------
+  struct Record
+  {
+    Record();
+    Record(float _E, /*float _w,*/ int _prod_id, const std::vector<Particle>& _ps);
+
+    /// Order by energy
+    bool operator<(const Record& rhs) const;
```

Comment:
This is surprising, do we really need to order this kind of objects?

Reply:
Added a comment.